### DISCUSSION TITLE
Unnecessary error showing on windows

### DISCUSSION THREAD

**Discussion by saidurraahmaan** [cpeditor/cpeditor#950] [Q&A] [answered]
I have installed LLVM and I have added MinGW to my env variable 
But see the screenshot. It detecting unnecessary errors. But my code compiled fine
Is there any way to fix this(windows 10 user)
![image](https://user-images.githubusercontent.com/54942548/127854360-48d6beac-6201-47c0-a81b-fe81c48a6bfb.png)

**Comment by ouuan** [comment, marked as answer]
Are you sure you are using the clangd bundled in CP Editor? Check you PATH environment variable and see if there are multiple paths that contain the clangd executable.

**Reply by saidurraahmaan** [reply]
can you elaborate? i didn't get u

**Reply by coder3101** [reply]
> can you elaborate? i didn't get u

TLDR; what's the path of clangd in Language Server settings of cpeditor.

**Reply by ouuan** [reply]
> can you elaborate? i didn't get u

Do you know what the PATH environment variable is? I don't know how to elaborate if you don't tell us which part you don't understand.

**Reply by saidurraahmaan** [reply]
> > can you elaborate? i didn't get u
> 
> TLDR; what's the path of clangd in Language Server settings of cpeditor.

see the screenshot. its clangd

**Reply by coder3101** [reply]
Yes, it's your custom install of LLVM, uninstall it and install any cpeditor beta version with LLVM for your PC and put the path of clangd from cpeditor.

It should be like (not exactly but you get the point, the clangd inside cpeditor installation directory)  `C://ProgramFiles/cpeditor/llvm/bin/clangd`

**Reply by saidurraahmaan** [reply]
ok. I just download 6.9.2 with LLVM. Now it's completely ok. Thank you very much.
